src/shogun/transformer/Transformer.cpp

```diff
@@ -0,0 +1,20 @@
+#include <shogun/base/Parameter.h>
+#include <shogun/lib/exception/NotFittedException.h>
+#include <shogun/transformer/Transformer.h>
+
+namespace shogun
+{
+
+	CTransformer::CTransformer() : CSGObject()
+	{
+		SG_ADD(
+		    &m_fitted, "fitted", "Whether the transformer has been fitted.",
+		    MS_NOT_AVAILABLE);
+	}
+
+	void CTransformer::check_fitted() const
```

Comment:
i would rather go with the java like: `is_fitted` or `is_fit` depending which english we'd like to go with :) but let's stick to british (non north american english as in case of the exception type naming)